src/main/io/msp.c

```diff
@@ -341,6 +341,8 @@ static uint32_t packFlightModeFlags(void)
             continue;                 // this flightmode is not copied
         if(FLIGHT_MODE(1 << i))
             boxEnabledMask |= 1 << flightMode_boxId_map[i];
+        if (rcModeIsActive(BOXAIRMODE) && flightMode_boxId_map[i] == BOXAIRMODE)
+            boxEnabledMask |= 1 << flightMode_boxId_map[i];
```

Comment:
Just add `MB(BOXAIRMODE)` to `rcModeCopyMask`. That's all it takes to pack new flag ..